bower.json

```diff
@@ -33,6 +33,6 @@
     "tests"
   ],
   "dependencies": {
-    "webrtc-adapter": "7.7.0"
+    "webrtc-adapter": "8.0.0"
```

Comment:
Did you test this too? I remember bower having issues because the version available online wasn't available there too.

Reply:
Just running `npm install` within the janus directory, but worked as expected.